snipssonos/services/injection_service.py

```diff
@@ -0,0 +1,38 @@
+import json
+
+from snipssonos.helpers.mqtt_client import MqttClient
+
+
+class InjectEntitiesService:
+    MQTT_TOPIC_INJECT = 'hermes/asr/inject'
+
+    def __init__(self, hermes_host):
+        self.mqtt_client = MqttClient(hermes_host)
+        self.mqtt_client.run()
```

Comment:
if hermes would be able to publish to arbitrary topics...otherwise no ideas on top of mind, could you make a suggestion or just commenting for a future refactor (on some other PR)?

Reply:
Haha yes ! You're not the first one to request this for hermes :) ! I'll think about it! 
In the meantime, I'll change this in a refactor ! Maybe we can open an issue on the repo to not forget about it !